apps/trading/client-pages/market/market.tsx

```diff
@@ -134,3 +123,15 @@ export const MarketPage = () => {
     </AsyncRenderer>
   );
 };
+
+const getAsset = (market: MarketFieldsFragment | null) => {
+  return market &&
```

Comment:
I would find the asset by using a products typename. This will set us up better for other spot and other product types